log/formatters.go

```diff
@@ -64,3 +65,61 @@ func shortPath(pathIn string) string {
 
 	return strings.Join(resultToks, string(filepath.Separator))
 }
+
+type JSONFormatter struct{}
+
+var DefaultJSONFieldKeys = []string{"level", "timestamp", "caller", "message"}
+
+func (jf *JSONFormatter) Format(entry *logrus.Entry) ([]byte, error) {
+	data := make(logrus.Fields, len(entry.Data)+4)
+	for k, v := range entry.Data {
+		if strings.HasPrefix(k, "saaskit.") {
+			continue
+		}
+
+		switch v := v.(type) {
+		case error:
+			// Otherwise errors are ignored by `encoding/json`
+			// https://github.com/sirupsen/logrus/issues/137
+			data[k] = v.Error()
+		default:
+			data[k] = v
+		}
+	}
+
+	// Configure default fields.
+	prefixDefaultFieldClashes(data)
+	data["timestamp"] = entry.Time.Format("2006/01/02 15:04:05")
```

Comment:
Should we use this same time format or should we stick to something more JSON-y?

Reply:
this looks ok to me